.github/workflows/code_scan.yml

```diff
@@ -7,6 +7,28 @@ on:
     - cron: "0 18 * * 1-5"
 
 jobs:
+  Trivy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: python -m pip install tox
+      - name: Trivy Scanning
+        env:
+          TRIVY_DOWNLOAD_URL: ${{ vars.TRIVY_DOWNLOAD_URL }}
```

Comment:
Should this URL be retrieved in the variable? Why not just be specified here? I guess that URL has a version number, and it is the reason to write it to the source file (`.yml`).

Reply:
@vinnamkim because I don't want to update worflow file itself when there is a new release of the trivy tool. we can just update the download URL on the variables.